components/brave_sync/crypto/crypto_unittest.cc

```diff
@@ -239,6 +241,9 @@ TEST(CryptoTest, Passphrase) {
       "mountain jelly";
   EXPECT_TRUE(PassphraseToBytes32(bip_passphrase, &bip_bytes));
   EXPECT_EQ(PassphraseFromBytes32(bip_bytes), bip_passphrase);
+  EXPECT_TRUE(IsPassphraseValid(bip_passphrase));
+  EXPECT_FALSE(IsPassphraseValid(""));
+  EXPECT_FALSE(IsPassphraseValid(bip_passphrase + " something wrong"));
```

Comment:
One more case I would suggest is:
```
const std::string bip_invalid_passphrase =
      "magic vacuum wide review love peace century egg burden clutch heart "
      "cycle annual mixed pink awesome extra client cry brisk priority maple "
      "mountain brave";
EXPECT_FALSE(IsPassphraseValid(bip_invalid_passphrase));
```
(I kept the length the same, but replaced the last word with an invalid word -- `brave`.)

Reply:
thanks @fmarier , updated in https://github.com/brave/brave-core/pull/10960/commits/69caa7fe0b3566c3f743291a7ac2509ae17c87c3#diff-b912644143e341afc9dc952720124c3876ff375bb276ca70f9c90346c474759fR242